checker/jdk/index/src/java/lang/Object.java

```diff
@@ -211,7 +211,7 @@ public boolean equals(Object obj) {
      *               be cloned.
      * @see java.lang.Cloneable
      */
-    protected native Object clone() throws CloneNotSupportedException;
+    protected native @PolySameLen Object clone(@PolySameLen Object this) throws CloneNotSupportedException;
```

Comment:
I added this because a test was failing without it; I don't remember which one. Either way, I think leaving this as an explicit annotation makes what's occurring clearer, so even if it isn't entirely necessary I'd advocate keeping it.